new-integration-tests/src/test/java/oracle/weblogic/domain/PersistentVolume.java

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Looks like some confusion here.  We had already agreed to use the Java types from the Kubernetes Java client, which for PV's is io.kubernetes.client.openapi.models.V1PersistentVolume.  There are similar classes for all of the other standard Kubernetes types.  We only created types for Domain because, of course, this doesn't exist in the standard set.

Reply:
Deleted the new types